pkg/memory-monitor/build.yml

```diff
@@ -0,0 +1,18 @@
+image: eve-memory-monitor
+org: lfedge
+config:
+  binds:
+    - /persist:/persist # for the working directory and output directory
+    - /dev:/dev # for syslog (/dev/log)
+    - /run:/run:ro # for the zedbox pid file
+    - /sys/fs/cgroup/memory/:/sys/fs/cgroup/memory/ # to read and modify memory limits
+    - /proc:/proc #:ro to read memory usage in /proc/<PID> and RW for the eve command (???)
```

Comment:
what we just mount it as `ro`, do we care that all other `eve` commands work in the memory-monitor container?

Reply:
I attempted to mount this as read-only, but the commands in my script didn't work. I can't recall the specifics, but it seemed to me that /proc needed to be read-write for ANY `eve` run.

Reply:
I checked and figured out that RW access was necessary only for the `eve exec` command. I reworked the image, and now I can run the necessary tools from the memory-monitor container without running this command. So, I removed RW access.